### ISSUE TITLE
Ambitus per voice

### ISSUE THREAD

**Issue by iainhallam** [musescore/MuseScore#18942]
### Your idea

Having added an ambitus to a clef, allow the user to select in properties whether it applies to all voices in a staff or a subset of voices (one possibility would be checkboxes for each voice, which default to all being on). The user would also be able to somehow add extra ambiti for the other voices.

### Problem to be solved

In choral music, it's often useful to know what the vocal range of each voice on the staff is in closed score, rather than the range for all singers on the staff. For instance, if Soprano and Alto parts are notated as voices on a single staff, the ambiti should show separate ranges for each.

### Additional context

In order to distinguish between the ambiti, perhaps the short part names should automatically be shown as text above the ambitus for each voice?

**Comment by mmeyn** [comment]
I don’t think you’ll need the short part names above the voices automatically: In most cases the higher voice or first player (soprano, flute 1, …) will have the higher ambitus. Also, the order of the ambitus from left to right should match the instrument/voice order from top to bottom so I don’t see a need for such texts.

One could have it as an option (default off) but I’ve never seen it anywhere.

**Comment by iainhallam** [comment]
>> In order to distinguish between the ambiti, perhaps the short part names should automatically be shown as text above the ambitus for each voice?

> I don’t think you’ll need the short part names above the voices automatically: In most cases the higher voice or first player (soprano, flute 1, …) will have the higher ambitus. Also, the order of the ambitus from left to right should match the instrument/voice order from top to bottom so I don’t see a need for such texts.

You're right about that, and in the few cases where it might be ambiguous, a text object would be perfectly feasible. I'll remove that suggestion from the request!
